consumer-test/consumerTestController.groovy

```diff
@@ -0,0 +1,213 @@
+import groovy.io.FileType
+
+import static groovy.json.JsonOutput.toJson
+
+COMMIT_HASH = null
+RUNNING_LOCALLY = false
+AUXILIARY_SLEEP_MS = 10000
+// Build is killed at 50 min, print log to console at minute 45
+PRINT_LOGS_AFTER_45_MINUTES_COUNTDOWN = (45 * 60 * 1000) / AUXILIARY_SLEEP_MS
+WORKSPACES_ROOT = 'workspaces'
+TEST_CASES_DIR = 'testCases'
+LIBRARY_VERSION_UNDER_TEST = null
+
+EXCLUDED_FROM_CONSUMER_TESTING_REGEXES = [
+    /^documentation\/.*$/,
+    /^.travis.yml$/,
+    /^test\/.*$/
+]
+
+
+if (!System.getenv('CX_INFRA_IT_CF_USERNAME') || !System.getenv('CX_INFRA_IT_CF_PASSWORD')) {
+    throw new RuntimeException('Environment variables CX_INFRA_IT_CF_USERNAME and CX_INFRA_IT_CF_PASSWORD need to be set.')
```

Comment:
Exceptions are not enough to make the travis build fail, the return code needs to be set to a positive value explicitly. So we need a System.exit(1) here.